vsintegration/src/FSharp.Editor/QuickInfo/QuickInfoProvider.fs

```diff
@@ -160,12 +160,15 @@ module private FSharpQuickInfo =
 type internal FSharpAsyncQuickInfoSource
     (
         serviceProvider: IServiceProvider,
-        xmlMemberIndexService: IVsXMLMemberIndexService,
         checkerProvider:FSharpCheckerProvider,
         projectInfoManager:FSharpProjectOptionsManager,
         textBuffer:ITextBuffer
     ) =
 
+    // GetService calls must be made on the UI thread
```

Comment:
It's unclear _why_ the following lines are always called from the UI thread though.

Reply:
Yes, I'll move it to TryCreateQuickInfoSource which should be safe according to https://github.com/Microsoft/visualfsharp/pull/4713#issuecomment-380566134